packages/admin/src/providers/dataProvider.js

```diff
@@ -26,28 +26,17 @@ export default {
 	getList: (resource, params) => {
 		const { page, perPage } = params.pagination;
 		const { field, order } = params.sort;
-		const query = {
+		let query = {
 			page,
 			perPage,
 			order,
 			orderBy: field,
 		};
+		query = params.embed ? { ...query, embed: '' } : query;
+
```

Comment:
Assim funciona tbm e fica mais organizado

```suggestion
		const query = {
			page,
			perPage,
			order,
			orderBy: field,
			embed: params.embed ? '' : undefined,
		};

```

Reply:
Feito!